sdk/keyvault/Azure.ResourceManager.KeyVault/api/Azure.ResourceManager.KeyVault.netstandard2.0.cs

```diff
@@ -797,6 +913,8 @@ public partial class ManagedHsmNetworkRuleSet
     public partial class ManagedHsmPrivateEndpointConnectionItemData
     {
         internal ManagedHsmPrivateEndpointConnectionItemData() { }
+        public Azure.ETag? ETag { get { throw null; } }
+        public string Id { get { throw null; } }
```

Comment:
Do we know if this is an arm-id? If so, we should change the format here

Reply:
According to the [example](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2023-02-01/examples/ManagedHsm_putPrivateEndpointConnection.json#L20), it is an `arm-id`